embedded_sass.proto

```diff
@@ -116,13 +116,17 @@ message InboundMessage {
       // including scheme.
       string url = 2;
 
-      // A path on disk to look for the file to load. The host must resolve
+      // A path on disk to look for the file to load. The compiler must resolve
       // partials, file extensions, and index files to determine whether a file
       // actually exists and what its canonical URL is.
       //
       // This response exists as a shorthand for importers that just encode
       // logic for finding files, so they don't have to re-implement all of
       // Sass's resolution logic.
+      //
+      // If an importer returns this reponse, it must accept also
```

Comment:
`accept also` -> `also accept`

Reply:
I'm just dropping this paragraph entirely since we're getting rid of this anyway.